replace old ddg with mine, also closes #17

adedomin opened the pull request.

adedomin committed:
Fix url parser and dockerfile

adedomin committed:
fix masshl logged exception

adedomin committed:
replace old ddg with mine, also closes #17

adedomin commented:
why didn't this....... ugh it links to FrozenPig's wtf.

ineeee commented on plugins/duckduckgo.py:
```diff
@@ -1,185 +1,165 @@
+
+
+@hook.command('ddg', autohelp=False)
+def main(inp: str, say: Callable):
```
can u make the command `ddg` please? i think people wont use `.main`

(yea the function name is the command)

ineeee replied:
i realize you put the command in the decorator... either way this will add a `.main` command

ineeee commented on plugins/duckduckgo.py:
```diff
@@ -1,185 +1,165 @@
+
+@hook.command('ddg', autohelp=False)
+def main(inp: str, say: Callable):
+    """Entrypoint."""
```
the first docstring in a function decorated by `hook.command` is used by taigabot when people use `.commands`, `.help` and the command with no arguments (in this case `.main`) so these three will show `Entrypoint.` in the chat

Commit:
remove '=None' from @hook.command (part 3)

Commit:
remove '=None' from @hook.command (part 4)

according to:

    $ rg -A2 -F --max-depth 1 "@hook.command" | grep -E 'def(.*)='

i am done

adedomin committed:
ok

adedomin committed:
fix help

ineeee commented:
nice maymay my dude, thats lite tbh fam xD :fire_emoji: :one_hundred_emoji_underlined_twice_in_red:

ineeee commented:
i don't know how to close requested changes

ineeee reviewed (commented):
edit: yea i dont know how to stop it from saying i requested changes
